### PR TITLE
add comands: init, open, reset

### PR DESCRIPTION
**Pull request by al-kau**

### CODE REVIEW

File: src/Eppie.CLI/Eppie.CLI/UserInteraction/MainMenu.cs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        {
+            _menuCommand = menuCommand;
+            _hostApplicationLifetime = hostApplicationLifetime;
+        }
+
+        protected override Task DoWorkAsync(CancellationToken stoppingToken)
+        {
+            _cancellationTokenSource.Cancel();
+
+            Task.Run(async () =>
```
Comment (anatolse) [inline src/Eppie.CLI/Eppie.CLI/UserInteraction/MainMenu.cs:50]: I didn't get what is happening here

File: src/Eppie.CLI/Eppie.CLI/UserInteraction/ConsoleElement.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+//   distributed under the License is distributed on an "AS IS" BASIS,          //
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   //
+//   See the License for the specific language governing permissions and        //
+//   limitations under the License.                                             //
+//                                                                              //
+// ---------------------------------------------------------------------------- //
+
+using Tuvi.Toolkit.Cli;
+
+namespace Eppie.CLI.UserInteraction
```
Comment (anatolse) [inline src/Eppie.CLI/Eppie.CLI/UserInteraction/ConsoleElement.cs:21]: CLI is already UserInteraction

File: src/Eppie.CLI/Eppie.CLI/Services/WorkService.cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+//   limitations under the License.                                             //
+//                                                                              //
+// ---------------------------------------------------------------------------- //
+
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Eppie.CLI.Services
+{
+    internal abstract class WorkService : BackgroundService
```
Comment (anatolse) [inline src/Eppie.CLI/Eppie.CLI/Services/WorkService.cs:24]: What is this class for?

File: src/Eppie.CLI/Eppie.CLI/UserInteraction/ConsoleElement.cs (hunk 2 above)
Comment (al-kau) [reply src/Eppie.CLI/Eppie.CLI/UserInteraction/ConsoleElement.cs]: renamed

File: src/Eppie.CLI/Eppie.CLI/Services/WorkService.cs (hunk 3 above)
Comment (al-kau) [reply src/Eppie.CLI/Eppie.CLI/Services/WorkService.cs]: removed

File: src/Eppie.CLI/Eppie.CLI/UserInteraction/MainMenu.cs (hunk 1 above)
Comment (al-kau) [reply src/Eppie.CLI/Eppie.CLI/UserInteraction/MainMenu.cs]: changed

**Review by ValeraFinebits — APPROVED** [review]

**Review by anatolse — APPROVED** [review]

**Pull request merged**